wpimath/src/main/native/include/frc/trajectory/constraint/RectangularRegionConstraint.h

```diff
@@ -29,14 +33,49 @@ class RectangularRegionConstraint : public TrajectoryConstraint {
    */
   RectangularRegionConstraint(const Translation2d& bottomLeftPoint,
                               const Translation2d& topRightPoint,
-                              const TrajectoryConstraint& constraint);
+                              Constraint constraint)
+      : m_bottomLeftPoint(bottomLeftPoint),
+        m_topRightPoint(topRightPoint),
+        m_constraint(std::make_unique<Constraint>(constraint)) {}
+
+  RectangularRegionConstraint(const RectangularRegionConstraint& rhs)
+      : TrajectoryConstraint() {
```

Comment:
Why is the explicit base class ctor call needed?

Reply:
gcc errors if that doesn't exist

Reply:
Shouldn't we be calling `TrajectoryConstraint(rhs)` instead?